ncm-afsclt/src/main/perl/afsclt.pm

```diff
@@ -106,34 +111,36 @@ sub Configure_Cache {
         $self->warn("Cannot determine current AFS cache size, changing only config file");
     }
 
-    my $afs_cacheinfo_fh = CAF::FileEditor->new( $AFS_CACHEINFO, log => $self ); # TODO use FileReader
-    $afs_cacheinfo_fh->cancel();
-    if ( "$afs_cacheinfo_fh" =~ m;^([^:]+):([^:]+):(\d+)$; ) {
+    my $afs_cacheinfo_fh = CAF::FileReader->new( $AFS_CACHEINFO, log => $self );
+    $self->debug(2, "$AFS_CACHEINFO current contents: >>>$afs_cacheinfo_fh<<<");
```

Comment:
In fact, I realized that my mod was wrong because if was no longer possible to override value in `cacheinfo` by the Quattor configuration. `cacheinfo` should only be used as default values. fixed.